spec/rubocop/cop/rails/presence_spec.rb

```diff
@@ -67,4 +67,13 @@
       a.blank? ? nil : b
     RUBY
   end
+
+  it 'does not register an offense when if or unless modifier is used ' do
+    [
+      'foo if foo.present?',
```

Comment:
In this case, I think it should register an offense for this code...

Reply:
I tried to register this pattern as offense, but I could not get it right. Could somebody help me?

Reply:
Probably we should rewrite the `redundant_receiver_and_other` and/or `redundant_negative_receiver_and_other` patterns. Let's try to rewrite the patterns with [a documentation of Node Pattern](https://github.com/bbatsov/rubocop/blob/master/manual/node_pattern.md), or replace the `expect_no_offenses` with `expect_offense` and make pending the test case.

Reply:
Thank you for your advice. I didn't have to rewrite neither `redundant_receiver_and_other` or `redundant_negative_receiver_and_other`. What I did was that adding `a unless a.blank?` to offense.